### ISSUE TITLE
Stuck on loading loop and not showing the website properly

### ISSUE THREAD

**Issue by RayGirl0712** [LiveSplit/LiveSplitOne#559]
I have used the website on Safari (iPhone 6s) a quite while until it has been stuck on the loading screen. No matter how many times I tried to fix but it won’t load properly.

**Comment by CryZe** [comment]
Yeah I noticed that it doesn't show the error message it's supposed to show. Basically we require a more modern browser now. Make sure to update your iOS version.

**Comment by CryZe** [comment]
During some refactoring recently I've seen that our try catch is not around the awaits of the individual modules, so those likely fail and the exception is uncaught. So this should be an easy fix.

**Comment by nlongname** [comment]
I've been having this problem on fully-updated Brave on Linux ("Version 93.1.29.76 (Official Build) (64-bit)") the last week or so (was working fine for weeks before that). Is that likely the same issue, or something different?

**Comment by CryZe** [comment]
Brave seems to be based on Chromium, which shouldn't have any problems. Can you check the dev tools if it shows any error messages?
